Incomplete TypeScript typings in @graphiql/toolkit@0.3.0

In version 0.3.0, `@graphiql/toolkit` dropped the dependency on `subscriptions-transport-ws`, making it a devDependency only. The problem is that `@graphiql/toolkit/dist/create-fetcher/types.d.ts` still performs a type import from `subscriptions-transport-ws`. This means that any TypeScript project using `@graphiql/toolkit` (or `graphiql`) but not `subscriptions-transport-ws` will now get a TypeScript error:

> node_modules/@graphiql/toolkit/dist/create-fetcher/types.d.ts:3:41 - error TS2307: Cannot find module 'subscriptions-transport-ws' or its corresponding type declarations.

--- a/.changeset/blue-eggs-tease.md
+++ b/.changeset/blue-eggs-tease.md
@@ -1,0 +1,6 @@
+---
+'graphiql': patch
+'@graphiql/toolkit': patch
+---
+
+Remove type definition from `subscriptions-transport-ws`

--- a/packages/graphiql-toolkit/src/create-fetcher/lib.ts
+++ b/packages/graphiql-toolkit/src/create-fetcher/lib.ts
@@ -1,7 +1,6 @@
 import { DocumentNode, visit, GraphQLError } from 'graphql';
 import { meros } from 'meros';
 import { createClient, Client, ClientOptions } from 'graphql-ws';
-import { SubscriptionClient } from 'subscriptions-transport-ws';
 import {
   isAsyncIterable,
   makeAsyncIterableIteratorFromSink,
@@ -116,9 +115,9 @@ export const createWebsocketsFetcherFromClient = (wsClient: Client) => (
     }),
   );
 
-export const createLegacyWebsocketsFetcher = (
-  legacyWsClient: SubscriptionClient,
-) => (graphQLParams: FetcherParams) => {
+export const createLegacyWebsocketsFetcher = (legacyWsClient: {
+  request: (params: FetcherParams) => unknown;
+}) => (graphQLParams: FetcherParams) => {
   const observable = legacyWsClient.request(graphQLParams);
   return makeAsyncIterableIteratorFromSink<FetcherResult>(
     // @ts-ignore

